app/scripts/BedLikeTrack.js

```diff
@@ -1109,7 +1113,58 @@ class BedLikeTrack extends HorizontalTiled1DPixiTrack {
         if (pc === -1) {
           const parts = visibleRects[i][1].value.fields;
 
-          return parts.join(' ');
+          let output = `<div class="track-mouseover-menu-table">`;
+
+          const identifierText = parts.length >= 4 ? parts[3] : null;
```

Comment:
Add comment describing this logic?

Reply:
I added a comment here and also to explain how the `intervalText` and `scoreText` variables are constructed.